.github/workflows/testistioadapter.yaml

```diff
@@ -0,0 +1,130 @@
+name: Test Istio Adapter
+
+on:
+  push:
+    branches:
+      - "*"
+    tags:
+      - "v*"
+  pull_request:
```

Comment:
@Revolyssup try changing this to `pull_request_target`
https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#pull_request_target